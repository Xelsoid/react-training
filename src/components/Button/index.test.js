import React from 'react';
import ComponentExample from './index';

describe('Button test', () => {
  it('renders correctly', () => {
    const component = shallow(<ComponentExample />);
    expect(component).toMatchSnapshot();
  });
});
