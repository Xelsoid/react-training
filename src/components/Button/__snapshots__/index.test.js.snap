// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button test renders correctly 1`] = `
<button
  className="btn btn--large"
  type="button"
>
  SEARCH
</button>
`;
